crates/goose/src/providers/model_pricing.rs

```diff
@@ -0,0 +1,125 @@
+use std::collections::HashMap;
+
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+use super::base::{Pricing, Usage};
+
+lazy_static::lazy_static! {
+    static ref MODEL_PRICING: HashMap<String, Pricing> = {
```

Comment:
this overlaps with ModelConfig in the https://github.com/block/goose/pull/466 a bit. Might be worthwhile to merge them

Reply:
Will get them both merged and look for opportunities.